src/canvas.ts

```diff
@@ -53,14 +28,14 @@ function getTextAlpha() {
  * Draws the background
  */
 function drawSpace() {
-  ctx.fillStyle = 'black';
-  ctx.fillRect(0, 0, cvs.width, cvs.height);
+  CTX.fillStyle = 'black';
+  CTX.fillRect(0, 0, CVS.width, CVS.height);
 }
 
 // function drawTriangle(
-//     use_center=true,
+//     use_CENTER=true,
 //     location=none,
-//     strokeColor='white',
+//     strokeColor='white',å
```

Comment:
a strange symbol in a strange comment btw

Reply:
Agreed. I think I started working on making the drawShip code more abstract (draw Triangle), but I never got around to it. I removed that commented-out block for now.